core/src/main/java/org/apache/iceberg/MergingSnapshotProducer.java

```diff
@@ -317,6 +319,18 @@ private ManifestFile copyManifest(ManifestFile manifest) {
     }
   }
 
+  @Override
+  public UpdateEvent updateEvent() {
+    return new CreateSnapshotEvent(
```

Comment:
I think this should also include the snapshot ID and sequence number that were committed.